### DISCUSSION TITLE
NZXT Kraken 2023 (Non X or Z model)

### DISCUSSION THREAD

**Discussion by matooch** [liquidctl/liquidctl#628] [Ideas]
Does anyone know if there will be support for the newest model of the Kraken? It seems like it is recognized as an USB device, but liquidctl does not see the device or is able to control it. Thanks!

**Comment by aleksamagicka** [comment]
Should be related: #605.

**Reply by matooch** [reply]
Oh amazing! Is this a separate branch?

**Reply by aleksamagicka** [reply]
Yes, in [a fork](https://github.com/brokenmass/liquidctl/tree/nzxt-kraken-elite). It's not merged yet, as evidenced by the PR itself.
